homeassistant/components/filesize/sensor.py

```diff
@@ -1,55 +1,44 @@
 """Sensor for monitoring the size of a file."""
 import datetime
-import logging
 import os
 
-import voluptuous as vol
-
-from homeassistant.components.sensor import PLATFORM_SCHEMA
-from homeassistant.const import DATA_MEGABYTES
-import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_FILE_PATH, CONF_UNIT_OF_MEASUREMENT, DATA_BYTES
 from homeassistant.helpers.entity import Entity
-from homeassistant.helpers.reload import setup_reload_service
-
-from . import DOMAIN, PLATFORMS
-
-_LOGGER = logging.getLogger(__name__)
+from homeassistant.util import slugify
+import homeassistant.util.data_size as data_size
 
+from .const import DOMAIN
 
-CONF_FILE_PATHS = "file_paths"
 ICON = "mdi:file"
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {vol.Required(CONF_FILE_PATHS): vol.All(cv.ensure_list, [cv.isfile])}
-)
-
-
-def setup_platform(hass, config, add_entities, discovery_info=None):
```

Comment:
Should we keep this around to allow importing it as config entry ?

Reply:
I have re-added and adjusted the method to the new config schema. Can you please check if okay.